src/pages/ProductList/components/FilterModal.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import './FilterModal.scss';
+
+export default function FilterModal({ isClicked }) {
+  const [selectedSize, setSelectedSize] = useState([]);
+  const selectedSizeList = [...selectedSize];
+
+  const clickSelectSize = event => {
+    const { name, type, checked } = event.target;
+    if (type === 'checkbox' && checked) {
+      setSelectedSize([...selectedSize, name]);
+    } else {
+      setSelectedSize(selectedSize.filter(size => size !== name));
+    }
+  };
+
+  return (
+    <form
+      className="filterModal"
+      style={isClicked ? { display: 'flex' } : { display: 'none' }}
```

Comment:
- 인라인 스타일 사용은 반드시 필요한 경우가 아니면 지양해주세요
1. 우선순위가 !important 제외하고 가장 높게 적용됨
2. css관련 로직이 css 파일이 아닌 html에 있어서 유지보수하기 좋지않음
- 위 이유들로 인해서 반드시 필요한 경우가 아닌 한 인라인 스타일은 지양해주세요

위 경우, 조건에 따라 다른 Ui를 보여주고 싶다면 인라인 스타일이 아니라, 어떤 속성을 이용하면 좋을까요?
더불어 위와 같은 경우 아래와 같이 작성해줄 수도 있습니다.
```suggestion
      style={{ display: isClicked ? 'flex' : 'none' }}
```

Reply:
네 신영님 인라인스타일에 대한 내용 확인했습니다.
```js
<form
      className={`filterDropDownWrapper ${isClicked ? 'show' : 'hide'}`}
```
sass 에서 show 와 hide 에 대한 display 값을 미리주고 클래스네임으로 핸들링하는 방법을 사용하는 방식으로 리팩토링 해봤어요!